qiskit/transpiler/passes/utils/gates_basis.py

```diff
@@ -0,0 +1,39 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Check if a property reached a fixed point."""
+
+from qiskit.transpiler.basepasses import AnalysisPass
+
+
+class GatesInBasis(AnalysisPass):
+    """Check if a a dag is all in the basis"""
+
+    def __init__(self, basis_gates):
+        """FixedPoint initializer.
```

Comment:
String needs to be updated.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/6832/commits/44635e25ab7ebf738fbeb228c315d55e7a4de071